tflint/config.go

```diff
@@ -377,6 +377,10 @@ func mergePluginMap(a, b map[string]*PluginConfig) map[string]*PluginConfig {
 	}
 	for k, v := range b {
 		ret[k] = v
+		if prevConfig, exists := ret[k]; exists && v.Body == nil {
+			// Use the plugin config from the config file when the plugin is enabled via CLI
+			ret[k].Body = prevConfig.Body
```

Comment:
It seems that `file` needs to be kept as well. When transferring the configuration to a plugin, it will be ignored if the file is nil.